.ci/gcb-generate-diffs.yml

```diff
@@ -1,8 +1,44 @@
 ---
 steps:
+    - name: 'alpine'
+      args:
+          - sh
+          - -c
+          - rm -rfv ./* ./.* || true
+    - name: 'alpine'
+      args:
+          - ls
```

Comment:
is this step just for debugging, or is it load bearing?

Reply:
Thanks!  Removed.